schedule/yast/ext4/ext4@yast.yaml

```diff
@@ -27,3 +27,13 @@ schedule:
   - installation/reboot_after_installation
   - installation/grub_test
   - installation/first_boot
+  - console/validate_ext4_fs
+test_data:
+  device: /dev/vda
+  partitions:
+    - mnt_point: /
+      fs_type: ext4
+    - mnt_point: /home
+      fs_type: xfs
+    - mnt_point: [SWAP]
+      fs_type: swap
```

Comment:
you can create a test data file and include it in both schedules using `!include` (check [docs](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/declarative-schedule-doc.md#test_data)

Reply:
Done